README.md

```diff
@@ -234,6 +237,14 @@ Otherwise, the `callback` function will be called with the following 3 arguments
 ### iterator#end(callback)
 <code>end()</code> is an instance method on an existing iterator object. The underlying LevelDB iterator will be deleted and the `callback` function will be called with no arguments if the operation is successful or with a single `error` argument if the operation failed for any reason.
 
+--------------------------------------------------------
+<a name="leveldown_snapshot"></a>
+### leveldown#snapshot([options])
+<code>snapshot()</code> is an instance method on an existing database object. It returns a new **Snapshot** instance. This can be passed into `get()`.
+
+#### `options`
```

Comment:
no point in adding options to the api if we don't have any yet, we can add that later if needed